packages/api/src/controllers/stream.js

```diff
@@ -322,6 +323,26 @@ app.put('/:id/setactive', authMiddleware({}), async (req, res) => {
   res.end()
 })
 
+app.put('/:id/record', authMiddleware({}), async (req, res) => {
+  const { id } = req.params
+  const stream = await req.store.get(`stream/${id}`, false)
+  if (!stream || stream.deleted || !req.user.admin) {
```

Comment:
Wait, won't this require that the user is an admin? `!req.user.admin` will always be false for regular users